com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -4279,7 +4279,8 @@ TextureHandle ColorGradingPass(RenderGraph renderGraph)
             var currentGradingHash = ComputeLUTHash();
 
             // The lut we have already is ok.
-            if (currentGradingHash == m_LutHash)
+            if (currentGradingHash == m_LutHash &&
```

Comment:
Does this mean that we recompute the LUT every frame whenever any parameter of the curve is overridden? I suppose this can actually happen quite often :/

Reply:
Yes, but the LUT is cheaper to compute than the hash (and up until really recently it was computed every frame anyhow :P )